build.sbt

```diff
@@ -69,6 +69,7 @@ val dataplug = project
       Library.Specs2.core,
       Library.Specs2.matcherExtra,
       Library.Specs2.mock,
+      Library.Utils.apacheCommonLang,
```

Comment:
why do we need this ?

Reply:
This is needed cause Java 11. It's what solved the compatibilities we had before. That means that the ugliness with the reflection remains

Reply:
so just adding this import fixes it ?

Reply:
Yes 👍

Reply:
The incompatibility is in this package. It turns out they have pushed an update that solves the errors. Gus found this solution